package.json

```diff
@@ -51,8 +51,9 @@
     "build": "babel --ignore=__tests__ src --out-dir lib",
     "lint": "standard",
     "prepublish": "npm test",
-    "posttest": "NODE_ENV=production npm run build",
+    "pretest": "npm run build",
```

Comment:
You're right, I'm getting mixed up here (I hate tooling :wink: ).

This `pretest` script is unneeded. The tests will catch a type error in the module, whether or not this `pretest` script is here (for some reason, I was thinking we were testing the compiled code, I don't know why).

However, regardless of how I configure it, it won't catch a type error of the tests calling the module with the wrong type.